aspnetcore/toc.yml

```diff
@@ -933,6 +933,9 @@
         - name: Transform web.config
           displayName: deploy, publish
           uid: host-and-deploy/iis/transform-webconfig
+        - name: HTTP/2
+          displayName: deploy, publish
+          uid: host-and-deploy/iis/iis-http2
```

Comment:
Eventually IIS will support HTTP/3. Should this page have a more generic ID so that content could be added.

`host-and-deploy/iis/iis-protocols`?